jobspec/parse.go

```diff
@@ -537,6 +538,22 @@ func parseTasks(jobName string, taskGroupName string, result *[]*structs.Task, l
 					return err
 				}
 			}
+
+			// Instantiate a driver to validate the configuration
+			d, err := driver.NewDriver(
```

Comment:
This will cause an import cycle (driver imports structs and vice-versa). Do you see any way around that?